### PR TITLE
Feature/pw 833 checkout

### PR DESCRIPTION
**Pull request by AlexandrosMor**
Added the following API calls:
/paymentSession
/payments/result
/originKeys

### CODE REVIEW

File: Adyen.EcommLibrary.Test/BaseTest.cs [hunk 1]
```diff
@@ -142,6 +142,28 @@ protected Model.Checkout.PaymentMethodsRequest CreatePaymentMethodRequest(string
...
+
+        /// <summary>
+        /// Checkout paymentsessionRequest
+        /// </summary>
+        /// <returns></returns>
+        protected Model.Checkout.PaymentSessionRequest CreatePaymentSessionRequest()
+        {
+            return new Model.Checkout.PaymentSessionRequest(MerchantAccount: "MerchantAccount", Reference: "MerchantReference",
+                 Amount: new Model.Checkout.Amount("EUR", 1200), ReturnUrl: @"https://your-company.com/...", CountryCode: "NL",
+                 Channel: Model.Checkout.PaymentSessionRequest.ChannelEnum.Web, SdkVersion: "1.3.0");
```
Comment (rikterbeek) [inline Adyen.EcommLibrary.Test/BaseTest.cs]: SdkVersion should not be needed for web right ?

File: Adyen.EcommLibrary.Test/CheckoutTest.cs [hunk 2]
```diff
@@ -3,21 +3,35 @@
...
 namespace Adyen.EcommLibrary.Test
 {
     [TestClass]
     public class CheckoutTest : BaseTest
     {
+        [TestMethod]
+        public void CheckoutEndpointTest()
+        {
+            var config = new Config();
+            var client = new Client(config);
```
Comment (rikterbeek) [inline Adyen.EcommLibrary.Test/CheckoutTest.cs]: Can we  use the same contractors as java ? https://docs.adyen.com/developers/development-resources/libraries#usingthelibrary1

File: Adyen.EcommLibrary.Test/BaseTest.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen.EcommLibrary.Test/BaseTest.cs]: It is required field for web as well

File: Adyen.EcommLibrary.Test/CheckoutTest.cs (hunk 2 above)
Comment (AlexandrosMor) [reply Adyen.EcommLibrary.Test/CheckoutTest.cs]: it is added at [commit](https://github.com/Adyen/adyen-dotnet-api-library/pull/39/commits/ccd837d91e7e190eeab5acd405d97e4e9b60ba3f)

File: Adyen.EcommLibrary.Test/BaseTest.cs (hunk 1 above)
Comment (rikterbeek) [reply Adyen.EcommLibrary.Test/BaseTest.cs]: ok, clear.

File: Adyen.EcommLibrary.Test/CheckoutTest.cs (hunk 2 above)
Comment (rikterbeek) [reply Adyen.EcommLibrary.Test/CheckoutTest.cs]: great!

**Pull request merged**